bio/dada2/filter-trim/environment.yaml

```
@@ -0,0 +1,7 @@
+channels:
+  - bioconda
+  - conda-forge
+  - defaults
+dependencies:
+  - bioconductor-dada2 = 1.16.0
```

We try to fix the exact version, rather use `==` instead of `=`.

```suggestion
dependencies:
  - bioconductor-dada2 ==1.16.0
```

---

Thanks! I did all your corrections in a single commit for clarity!